Week-4/Session 10 JS/S10_this.js

```diff
@@ -0,0 +1,37 @@
+class Person {
+  constructor(Name, Age, Gender, Nationality) {
```

Comment:
ideally the arguments shouldn't be capitalized. only class names in js are capitalized.

Reply:
refactor accordingly in this file.

Reply:
okay